commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -223,7 +222,11 @@ public String getMetricsNetworkInterface() {
 
   @Override
   public Optional<TlsOptions> getTlsOptions() {
-    return Optional.ofNullable(picoCliTlsServerOptions);
+    if (picoCliTlsServerOptions.getKeyStoreFile() != null
+        && picoCliTlsServerOptions.getKeyStorePasswordFile() != null) {
+      return Optional.of(picoCliTlsServerOptions);
+    }
```

Comment:
Might be nice if the picoCliTlsServerOptions returned a Optional<TlsOptions> so this logic could be moved out the base command and stay with the TLS options